Appium/Dockerfile

```diff
@@ -150,6 +150,7 @@ ENV APPIUM_DRIVER_ESPRESSO_VERSION="2.29.2"
 ENV APPIUM_DRIVER_FLUTTER_VERSION="2.2.2"
 ENV APPIUM_DRIVER_GECKO_VERSION="1.2.9"
 ENV APPIUM_DRIVER_UIAUTOMATOR2_VERSION="2.34.1"
+ENV ANDROID_ADB_SERVER_ADDRESS="host.docker.internal"
```

Comment:
we dont need to put it inside the Dockerfile, but we can pass the variable on the fly, cant we? as User use different on OS and it seems it is needed only for Windows OS. Maybe you can write it in readme that this environment variable is needed for Windows OS

Reply:
Okay, I will fix this. Additionally, I will explore more, as currently, it is only confirmed to work with the UIAutomator2 Driver.

Reply:
Deep in dive into the source code, I found that the  capability `appium:remoteAdbHost` or `appium:host` both works.  

Take UiAutomator2 driver as example:

The capability was used by [uiautomator2Opts.host]( https://github.com/appium/appium-uiautomator2-driver/blob/8b6c678f23be2217db747147802cf65fa0227909/lib/driver.ts#L580)

the `uiautomator2Opts.host` was used by [JWProxy]( https://github.com/appium/appium-uiautomator2-driver/blob/8b6c678f23be2217db747147802cf65fa0227909/lib/uiautomator2.js#L75)

  `JWProxy` use it to [make request to the uiautomator-server running on android](https://github.com/appium/appium/blob/9c0cb8690f27844138aadf7f37ee9acfc95389d7/packages/base-driver/lib/jsonwp-proxy/proxy.js#L131C82-L131C82).

Espresso driver use the same logic.

https://github.com/appium/appium-espresso-driver/blob/3872085e9b8d6056ef655e364754e0488be49f40/lib/driver.js#L512

Flutter driver directly use UiAutomator2 .

https://github.com/appium/appium-flutter-driver/blob/330ab4b9745b36d84c3865c5d171eeb03cc6d558/driver/lib/sessions/android.ts#L9